products/oneplus.md

```diff
@@ -0,0 +1,103 @@
+---
+title: OnePlus
+category: device
+tags: oneplus 1plus mobile-phone
+iconSlug: oneplus
+permalink: /oneplus
+versionCommand: "Settings -> About device"
+releasePolicyLink: https://community.oneplus.com/thread/1462181
+releaseColumn: false
+releaseDateColumn: true
+activeSupportColumn: Active Major Updates
+discontinuedColumn: true
+eolColumn: Security Updates
+customColumns:
+-   property: supportedAndroidVersions
+    position: after-release-column
+    label: Supported Android
+    description: Supported Android versions range
+    link: https://endoflife.date/android
+
+###### Supported Android versions ######
+# Because OnePlus is using community forum posts as official releases posts it is hard to parse the last Android/OxygenOS releases 
+# for all devices so I'm using the unofficial https://oxygenupdater.com/news/all/ that have been collecting all OnePlus OxygenOS announcements for several years and is an established and trusted way to get OP updates.
+# still not great but easier to search and parse than OnePlus Community and more up to date than GsmArena
+# There is also https://www.androidupdatetracker.com/policy/oneplus that is easier to parse and up to date but AFAIK is less established
```

Comment:
What I meant is that I can't attest to the seriousness of https://www.androidupdatetracker.com/ because I saw at least one mistake in their oneplus policy here https://www.androidupdatetracker.com/policy/oneplus regarding the Nord N series. I have no info into their research and the site is mentioned nowhere. Contrary to https://oxygenupdater.com/ that have correct data (so far) even helping me see mistakes GSMarena had done in release dates. I saw them mentioned several times in blogs, and they have an open source app on github that shows that they've been present for 4-5 years.

Reply:
I've moved these links to the description, because they might be helpful to end users as well. Didn't feel the nuance about Nord N was worth adding for now (Nord N isn't being tracked for now as well).